nbviewer/providers/local/handlers.py

```diff
@@ -144,7 +173,7 @@ def get(self, path):
                                    breadcrumbs=self.breadcrumbs(path),
                                    title=os.path.basename(path))
 
-    def show_dir(self,  abspath,  path):
+    def show_dir(self, abspath, path):
```

Comment:
If changing `abspath` to `fullpath` above, should the same be done here for consistency?

Reply:
Happy to do so.